web/styles/popovers.css

```diff
@@ -617,21 +617,32 @@ ul.popover-group-menu-member-list {
     .stream-list-top-section {
         display: flex;
         margin-top: 12px;
+        align-items: center;
+        margin-bottom: 5px;
 
-        .stream-search {
+        .stream-search-container {
             margin-left: auto;
-            align-self: center;
+            display: grid;
+            grid-template:
+                "search-input clear-search" auto / minmax(0, 1fr)
+                30px;
+        }
+
+        .stream-search {
+            grid-column: search-input-start / clear-search-end;
```

Comment:
For future reference, you can just specify `grid-area: search-input`; there's no need to indicate starts and ends, or have a separate value on `grid-row`.

Reply:
I tried this but it does not work because we want the "x" button column inside the input column. Using `grid-area: search-input` results in "x" button being outside the input to the right of it.